server/src/main/java/com/broadcom/lsp/cobol/LangServerBootstrap.java

```diff
@@ -78,30 +85,37 @@ private void start(
     }
   }
 
-  Launcher<LanguageClient> launchServer(@NonNull String[] args, @NonNull LanguageServer server)
+  Launcher<LanguageClient> launchServer(
+      @NonNull String[] args,
+      @NonNull LanguageServer server,
+      @NonNull ExecutorService executorService)
       throws IOException {
     return isPipeEnabled(args)
-        ? createServerLauncher(server, System.in, System.out)
-        : createServerLauncherWithSocket(server);
+        ? createServerLauncher(server, System.in, System.out, executorService)
```

Comment:
Any specific reason to start the server using this executor service?
If we don't, we can use customExecutor to stop all server processing by shutting down the customExecutor during shutdown call before exit.

Reply:
Yes, shutdown I forgot ;) thanks